packages/checkout-full/core/src/components/malga-checkout-full/malga-checkout-full.types.ts

```diff
@@ -192,19 +209,11 @@ export interface Product {
   risk: string
 }
 
-export interface Product {
-  name: string
-  amount: number
-  quantity: number
-  description: string
-  sku: string
-  risk: string
-}
-
```

Comment:
por que a gente pode remover essa interface?

Reply:
It seems like it might have been duplicated during a merge